modules/virt-fusion-access-san-release-updates.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-fusion-access-san-release-updates_{context}"]
+= IBM Fusion Access for SAN release updates
+
+This section provides release updates for IBM Fusion Access for SAN, including new features, bug fixes, and known issues.
+
+[id="virt-fusion-access-san-new-changes_{context}"]
+== New and changed features
+
+[id="ocpnas-213-image-registry-requirements_{context}"]
+=== Image registry requirements for kernel module management
+
+{IBMFusionFirst} uses the {product-title} image registry to manage the kernel module. Do not configure the registry to use `emptyDir` storage because it provides only temporary storage and is not suitable for production use. Configure {IBMFusionFirst} to use a different image registry by creating a ConfigMap and Secret after installing the operator and before creating the FusionAccess CR. (link:https://issues.redhat.com/browse/OCPNAS-213[*OCPNAS-213*])
+
+[id="virt-fusion-access-san-bug-fixes_{context}"]
+== Bug fixes
+
+[id="ocpnas-184-filesystem-creation-delay_{context}"]
+=== Filesystem creation button stays disabled until daemons are ready
+
+The {IBMFusionFirst} operator was updated to check the readiness of filesystem daemons before allowing a filesystem to be created. The **Create file system** button in the web console now stays disabled with a tooltip explaining the condition until the environment is ready. This change prevents filesystems from appearing stuck during creation. (link:https://issues.redhat.com/browse/OCPNAS-184[*OCPNAS-184*])
+
+[id="ocpnas-217-filesystem-deletion-ui_{context}"]
+=== Filesystems cannot be deleted from the user interface
+
+The web console does not support deleting filesystems. To delete a filesystem, use the CLI. (link:https://issues.redhat.com/browse/OCPNAS-217[*OCPNAS-217*])
+
+[id="virt-fusion-access-san-known-issues_{context}"]
+== Known issues
+
+[id="ocpnas-233-filesystem-creation-fails_{context}"]
+=== Filesystem creation might fail during core pod deletion
+
+Filesystem creation might fail if core pods are deleted at the same time. The filesystem might be partially created on the LUN, which results in the following persistent error:
+
+----
+Disk <ID> may still belong to an active file system
+----

```

Comment:
```suggestion
[source,terminal]
----
Disk <ID> may still belong to an active file system
----
```

See [guidelines on code blocks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output)